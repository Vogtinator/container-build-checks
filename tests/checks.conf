[General]
Vendor=org.opensuse
Registry=registry.opensuse.org
