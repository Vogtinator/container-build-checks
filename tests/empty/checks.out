Looking at empty.containerinfo
Release: 42.0
Warning: None of the tags are unique to a specific build of the image.
Make sure that at least one tag contains the release.
Error: org.openbuildservice.disturl not set correctly, bug in OBS?
Warning: Could not determine image specific label prefix, some checks will be skipped.
Warning: Label org.openbuildservice.disturl is not set by the image or any of its bases
Warning: Label org.opencontainers.image.title is not set by the image or any of its bases
Warning: Label org.opencontainers.image.description is not set by the image or any of its bases
Warning: Label org.opencontainers.image.version is not set by the image or any of its bases
Warning: Label org.opencontainers.image.created is not set by the image or any of its bases
Warning: Label org.opencontainers.image.vendor is not set by the image or any of its bases
Warning: Label org.opencontainers.image.url is not set by the image or any of its bases
Warning: Label org.opensuse.reference is not set by the image or any of its bases

container-build-checks done. Hints: 0 Warnings: 10 Errors: 1
Warnings found, but those are only fatal in certain projects.
Fatal errors found.
Exited with 1
